### PR TITLE
Update ArrayAnimatorWCS class

### PR DESCRIPTION
**Pull request by kathivashkiv**
Checks related to Astropy ver 4.0 have been removed
(#4000 issue)

### CODE REVIEW

**Comment by kathivashkiv** [comment]
@Cadair @nabobalis please have a look

File: sunpy/visualization/animator/wcs.py [hunk 1]
```diff
@@ -110,7 +103,10 @@ def _get_wcs_labels(self):
         Read first the axes names property of the wcs and fall back to physical types.
         """
         # world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.
-        world_axis_names = getattr(self.wcs, "world_axis_names", [''] * self.wcs.world_n_dim)
+        try:
+            world_axis_names = self.wcs.world_axis_names
+        except AttributeError:
+            world_axis_names = [''] * self.wcs.world_n_dim
```
Comment (nabobalis) [inline sunpy/visualization/animator/wcs.py:109]: I am not sure we need this.

File: sunpy/visualization/animator/wcs.py [hunk 2]
```diff
@@ -80,13 +80,6 @@ def __init__(self, data, wcs, slices, coord_params=None, ylim='dynamic', ylabel=
             image_axes.append(slices[::-1].index("y"))
             self.plot_dimensionality = 2
 
-        if self.plot_dimensionality == 1:
-            try:
-                from astropy.visualization.wcsaxes.frame import RectangularFrame1D  # NOQA
```
Comment (nabobalis) [inline sunpy/visualization/animator/wcs.py:85]: Do we still not need this import for something?

Comment (kathivashkiv) [reply sunpy/visualization/animator/wcs.py:85]: In `wcs.py` it is the only mention of it

Comment (nabobalis) [reply sunpy/visualization/animator/wcs.py:85]: Must have just been a check then.

File: sunpy/visualization/animator/wcs.py (hunk 1 above)
Comment (kathivashkiv) [reply sunpy/visualization/animator/wcs.py:109]: @Cadair said that this method can be rewritten not to use `getattr`, but I could have misunderstood the way to do it

Comment (nabobalis) [inline sunpy/visualization/animator/wcs.py:109]:
```suggestion
        world_axis_names = self.wcs.world_axis_names
```
I think this is all we need.

Comment (Cadair) [reply sunpy/visualization/animator/wcs.py:109]: I thought we still needed to generate the list of strings but [we don't](https://github.com/astropy/astropy/blob/master/astropy/wcs/wcsapi/low_level_api.py#L301) as that's the default in the base class. So @nabobalis's suggestion is correct.

File: sunpy/visualization/animator/wcs.py (hunk 2 above)
Comment (Cadair) [reply sunpy/visualization/animator/wcs.py:85]: yeah this was how I was checking if you were running an astropy version which supported 1D or not.

File: sunpy/visualization/animator/wcs.py [hunk 3]
```diff
@@ -110,7 +103,7 @@ def _get_wcs_labels(self):
         Read first the axes names property of the wcs and fall back to physical types.
         """
         # world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.
-        world_axis_names = getattr(self.wcs, "world_axis_names", [''] * self.wcs.world_n_dim)
+        world_axis_names = self.wcs.world_axis_names
         # Return the name if it is set, or the physical type if it is not.
         return [l or t for l, t in zip(world_axis_names, self.wcs.world_axis_physical_types)]
```
Comment (Cadair) [inline sunpy/visualization/animator/wcs.py:108]: Might as well drop the intermediate variable as well I think:
```suggestion
        world_axis_names = self.wcs.world_axis_names
        # Return the name if it is set, or the physical type if it is not.
        return [l or t for l, t in zip(self.wcs.world_axis_names, self.wcs.world_axis_physical_types)]
```

**Review by Cadair — APPROVED** [review]

File: sunpy/visualization/animator/wcs.py (hunk 3 above)
Comment (kathivashkiv) [reply sunpy/visualization/animator/wcs.py:108]: Then delete the first comment too?

Comment (kathivashkiv) [reply sunpy/visualization/animator/wcs.py:108]: `world_axis_names was only added to the APE 14 API in 4.0, so do this for backwards compatibility.`

Comment (dstansby) [reply sunpy/visualization/animator/wcs.py:108]: Yep, deleting the comment seems like the right thing to do

**Review by dstansby — APPROVED** [review]
Thanks for the PR 👍

**Pull request merged**